lib/common.sh

```diff
@@ -1,8 +1,23 @@
 #!/usr/bin/env bash
 
-CONFIG_FILE="$HOME/.stackrox/workflow-config.json"
+# Use CONFIG_HOME if it exists. Otherwise follow XDG base directory specification.
+CONFIG_HOME="${HOME}/.stackrox/workflow-config.json"
+if [ -f "${CONFIG_FILE}" ]; then
+	CONFIG_FILE=$CONFIG_HOME
```

Comment:
Quote RHS

Reply:
Why? There should be no value splitting in the assignment, I think quotes would be redundant. It's also not linted by shellcheck (as opposed to e.g. `ls $CONFIG_HOME` would be).

Reply:
I added the quotes anyways, just to make sure there is no confusion.

Reply:
Ah, just tried it out and you're right. Honestly the bash behaviour around this is so arcane, I feel it's better to just always quote (when that's what you want) rather than deciding based on whether quotes are necessary or not. But I guess just listening to `shellcheck` also works.